src/hotspot/share/runtime/globals.hpp

```diff
@@ -650,6 +650,10 @@ const int ObjectAlignmentInBytes = 8;
   develop(bool, StressCompiledExceptionHandlers, false,                     \
           "Exercise compiled exception handlers")                           \
                                                                             \
+  product(bool, DeoptimizeOnAllocationException, false, DIAGNOSTIC,          \
+          "Deoptimize on exception during allocation instead of using the"  \
+          " compiled exception handlers")                                   \
```

Comment:
For consistency with other flag definitions:
```suggestion
  product(bool, DeoptimizeOnAllocationException, false, DIAGNOSTIC,         \
          "Deoptimize on exception during allocation instead of using the " \
          "compiled exception handlers")                                    \
```

Reply:
Indeed, it was a bit ugly. I think I have my spaces in row now.